GetandTake/appsettings.json

```diff
@@ -1,21 +0,0 @@
-{
```

Comment:
I don't advise you to remove this file since it is used by default for application if no corresponding app settings file with environment was found (like `appsettings.Development.json`) and it makes easier to other devs to see the structure of file. So, let's leave it

Reply:
added user secrets and appsettings.json for this one